tests/unit/backend/chalice/api_server/test_v1_collection.py

```diff
@@ -190,3 +190,27 @@ def test__get_collection_uuid__403_not_found(self):
         response = self.app.get(test_url.url, headers=dict(host="localhost"))
         self.assertEqual(403, response.status_code)
         self.assertIn("X-AWS-REQUEST-ID", response.headers.keys())
+
+    def test__post_collection_returns_uuid_on_success(self):
+        test_url = furl(path="/dp/v1/collections/")
+        data = json.dumps({
+            "name": "bkjbjbjmbjm",
+            "description": "This is a test collection",
+            "contact_name": "person human",
+            "contact_email": "person@human.com",
+            "data_submission_policy_version": 1,
+            "links": [
+                {"name": "DOI Link", "url": "http://doi.org/10.1016", "type": "DOI"},
+                {"name": "DOI Link 2", "url": "http://doi.org/10.1017", "type": "DOI"}]
+        })
+        response = self.app.post(
+            test_url.url,
+            headers={"host": "localhost"},
```

Comment:
you need to add the `Content-Type` header. For example: https://github.com/HumanCellAtlas/fusillade/blob/b5ec9ed8fa3a45fc0f449ed01b828c2bbafc80a5/tests/test_user_api.py#L130

That way the backend knows how to convert it.

```suggestion
            headers={"host": "localhost", 'Content-Type': "application/json"},
```

Reply:
I keep get a 500 when I did that but now it magically works. Thank you!!